bindings/nodejs/lib/types/block/core/validation.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { StrongParents, WeakParents, ShallowLikeParents } from '../parents';
+import { HexEncodedString } from '../../utils';
+import { Block } from './block';
+
+/**
+ * A Validation Block is a special type of block used by validators to secure the network.
+ * It is recognised by the Congestion Control of the IOTA 2.0 protocol and can be issued without
+ * burning Mana within the constraints of the allowed validator throughput.
+ *
+ * It is allowed to reference more parent blocks than a normal Basic Block.
+ * Validation Block uses the introduced Block ID as its identifier.
```

Comment:
I think this sentence makes sense in the TIP but not here, I'd remove.

Reply:
which part specifically? the second part with It is.... its identifier?

Reply:
Yes, second sentence, we removed it in Rust already

Reply:
Sorry I mean second sentence of the second part! The last sentence.

Reply:
ill copy from rust 😆